CMakeLists.txt

```diff
@@ -82,6 +82,22 @@ if(NOT DEFINED PLUGIN_LIBRARY_OUTPUT_DIRECTORY)
 endif()
 message(STATUS "${CMAKE_PROJECT_NAME}: PLUGIN_LIBRARY_OUTPUT_DIRECTORY: ${PLUGIN_LIBRARY_OUTPUT_DIRECTORY}")
 
+# When building, don't use the install RPATH already
+# (but later on when installing)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+set(CMAKE_INSTALL_RPATH
+    "${CMAKE_INSTALL_PREFIX}/${PLUGIN_LIBRARY_OUTPUT_DIRECTORY};${CMAKE_INSTALL_PREFIX}/${PLUGIN_OUTPUT_DIRECTORY}"
```

Comment:
We can drop JANA_PLUGIN_PATH after that?

Reply:
I see. Looks like they don't do dlopen without a full path, so rpath can't apply. That is not very convenient, IMO.

Reply:
It might be useful to come up with a proposal as to how JANA2 should find/load plugins if/when `JANA_PLUGIN_PATH` is not set. One way could be to use the rpath/runpath of the current executable. Another may be requiring LD_PRELOAD or something.

In any case, I'm going to resolve this conversation.